runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3151,7 +3151,7 @@ void TR::CompilationInfo::stopCompilationThreads()
 
       TR::CodeCacheManager::instance()->printMccStats();
 
-      fprintf(stderr, "Allocated memory for data cache = %d KB\tLimit = %d KB\n",
+      fprintf(stderr, "Allocated memory for data cache = %d KB\tLimit = %lu KB\n",
```

Comment:
I realized that we have a cast to 32 bit here and therefore `l` is not needed, just `%u`

Reply:
I'd argue the format should be fixed instead of casting the data type, and it'd be more consistent with the fixes on the `master` branch as well. I've removed the cast in 2750c80.